src/tcgwars/logic/impl/gen3/PopSeries5.groovy

```diff
@@ -293,16 +251,27 @@ public enum PopSeries5 implements LogicCardInfo {
           attackRequirement {}
           onAttack {
             damage 10
+            flip  {
+              self.cards.energyCount(C), {damage 20}
```

Comment:
the count should be outside the closure so that flip executes that many times. Not sure how this will react as written.